Apply RFC Template Formatting

ve7jtb opened the pull request:
Apply RFC template formatting First pass

Needs more work but passes the check tools for publishing

ve7jtb committed:
Update draft-bradleylundberg-arkg-00.md

Apply RFC template formatting First pass

emlun commented on draft-bradleylundberg-arkg-00.md:
```diff
@@ -1,7 +1,63 @@
+###
+
+title: "The Asynchronous Remote Key Generation (ARKG) algorithm"
+abbrev: "AKRG"
+lang: en
+category: info
+
+docname: draft-bradleylundberg-cfrg-akrg
```
```suggestion
abbrev: "ARKG"
lang: en
category: info

docname: draft-bradleylundberg-cfrg-arkg
```

emlun reviewed (approved):
Thanks!

emlun committed:
Update draft-bradleylundberg-arkg-00.md

emlun committed:
Remove -00 from file name as instructed in top comment

ve7jtb committed:
Update draft-bradleylundberg-arkg.md

fix some references and add some formmating

ve7jtb committed:
Update draft-bradleylundberg-arkg.md

more fixed refrences

ve7jtb committed:
Update draft-bradleylundberg-arkg.md

I added the keywords boilerplate, Fixed some line length issues.  [Frymann] was a hanging refrence so I pointed them to [Frymann2020] please check that I pointed them to the correct one.

emlun commented on draft-bradleylundberg-arkg.md:
```diff
@@ -53,7 +53,69 @@ author:
+    author:
+    - name: Emil Lundberg
+    - name: Dain Nilsson
+    title: WebAuthn recovery extension Asynchronous delegated key generation without shared secrets. GitHub
```
The comma was dropped somehow?
```suggestion
    title: WebAuthn recovery extension: Asynchronous delegated key generation without shared secrets. GitHub
```

emlun replied:
Or in case quotes are needed for the comma to not confuse the parser:
```suggestion
    title: "WebAuthn recovery extension: Asynchronous delegated key generation without shared secrets. GitHub"
```

ve7jtb replied:
Done needed quotes to not mess up the kramdown parser.

emlun commented on draft-bradleylundberg-arkg.md:
```diff
@@ -53,7 +53,69 @@ author:
+    - name: Emil Lundberg
+    - name: Mark Manulis
+    - name: Dain Nilsson
+    title: Asynchronous Remote Key Generation - An Analysis of Yubico’s Proposal for W3C WebAuthn. CCS
```
Should this not include the rest of the name of the publication? Also colon instead of dash?

```suggestion
    title: "Asynchronous Remote Key Generation: An Analysis of Yubico’s Proposal for W3C WebAuthn. CCS '20: Proceedings of the 2020 ACM SIGSAC Conference on Computer and Communications Security"
```

ve7jtb replied:
done

emlun commented on draft-bradleylundberg-arkg.md:
```diff
@@ -53,7 +53,69 @@ author:
+    - name: Nick Frymann
+    - name: Daniel Gardham
+    - name: Mark Manulis
+    title: Asynchronous Remote Key Generation for Post-Quantum Cryptosystems from Lattices. 2023 IEEE 8th European Symposium on Security and Privacy,
```
```suggestion
    title: Asynchronous Remote Key Generation for Post-Quantum Cryptosystems from Lattices. 2023 IEEE 8th European Symposium on Security and Privacy
```

ve7jtb replied:
done

emlun commented on draft-bradleylundberg-arkg.md:
```diff
@@ -498,7 +510,7 @@ TODO: IANA registry? COSE/JOSE?
 
 Instantiations of ARKG whose output keys are elliptic curve keys
-can use elliptic curve arithmetic as the key blinding scheme `BL`. [Frymann] [Wilson]
+can use elliptic curve arithmetic as the key blinding scheme `BL`. [Frymann2020]
```
[Frymann] to [Frymann2020] is correct, thanks. But should we keep the [Wilson] reference here?

ve7jtb replied:
done
